### ISSUE TITLE
[Feature Request] Add TargetElement.focus property to Action.ToggleVisibility

### ISSUE THREAD

**Issue by Stevenic** [microsoft/AdaptiveCards#7922]
### Problem Statement

When visibility toggles are used there's an accessibility issue that focus is lost and there's no automatic way of establishing where focus should be moved to. 

### Proposed solution

I propose adding a new `focus` property to the `TargetElement` structure passed in to `Action.Togglevisibility`. I work a lot with visibility toggles and in my case the developer always knows where focus should be moved to when somethings visibility is toggled, There's no need in trying to automatically infer where visibility should be moved to just let the developer specify.

What happens if `focus` is set to true for more then one target element? Focus should move to the first one specified.
